examples/mnist/mnist.go

```diff
@@ -1,21 +1,23 @@
 package main
 
 import (
-	"fmt"
-
 	torch "github.com/wangkuiyi/gotorch"
 )
 
-func main() {
+func trainLoop() {
 	dataset := torch.NewMNIST("./data")
 	dataset.AddTransforms([]torch.Transform{
 		torch.NewNormalize(0.1307, 0.3081),
 		torch.NewStack(),
 	})
 	trainLoader := torch.NewDataLoader(dataset, 8)
 	for trainLoader.Scan() {
-		data := trainLoader.Data()
-		fmt.Println(data.Data)
-		fmt.Println(data.Target)
+		torch.GC()
+		trainLoader.Data()
+		torch.FinishGC()
```

Comment:
Now `torch.FinishGC()` should be called after the loop, rather than in the loop.